components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -170,16 +172,18 @@ ConversationDriver::GetVisibleConversationHistory() {
       list.push_back(turn.Clone());
```

Comment:
do we use conversation turn's visibility anywhere useful atm? iirc, this was introduced to hide internal prompts from the user.

Reply:
no